warehouse/templates/packaging/detail.html

```diff
@@ -35,7 +35,7 @@
   {% set icon = "fab fa-gitlab" %}
 {% elif parsed.netloc == "gitter.im" or parsed.netloc.endswith(".gitter.im") %}
   {% set icon = "fab fa-gitter" %}
-{% elif parsed.netloc in ["discord.com", "discordapp.com", "discord.gg"] or parsed.netloc.endswith(("discord.com", "discordapp.com", "discord.gg")) %}
+{% elif parsed.netloc in ["discord.com", "discordapp.com", "discord.gg"] or parsed.netloc.endswith((".discord.com", ".discordapp.com", ".discord.gg")) %}
```

Comment:
FLAG: I think the missing dot was an oversight but I'm unsure if Discord uses custom subdomains.

Reply:
Good catch, but. I don't think Discord uses subdomains like this, Discord URLs look like:

* `https://discord.com/channels/...`
* `https://discord.com/invite/...`
* `https://discordapp.com/channels/...`
* `https://discordapp.com/invite/...`
* `https://discord.gg/...`

```suggestion
{% elif parsed.netloc in ["discord.com", "discordapp.com", "discord.gg"] %}
```